### PR TITLE
Remove module deactivation from post-run hook

### PR DESCRIPTION
**Pull request by rwx788**
As a workaround for cnf we activate desktop-applications module, and
tried to deactivate it afterwards to bring the SUT to original state.
However, we have bigger problem in our flow. We use same VM image in
many tests, but that image shares same SCC credentials, so
activation/deactivation in one scenario will affect other scenarios.

For example, on aarch64 some test suite activated dev tools module, so
when we tried to deactivate desktop-applications, SCC correctly
responded that we should deactivate dev-tools module first, which is not
activated in this particular SUT. So, as activation still works fine, we
leave this soft-failure, but remove cleanup with module deactivation.

See [bsc#1079848](https://bugzilla.suse.com/show_bug.cgi?id=1079848).
[poo#31462](https://progress.opensuse.org/issues/31462).

Verification runs:
[sle15](http://g226.suse.de/tests/740)
[openSUSE](http://g226.suse.de/tests/742)

### CODE REVIEW

File: tests/console/command_not_found.pm [hunk 1]
```diff
@@ -33,34 +33,25 @@ sub run {
...
-        else {
-            die "Command Not Found failed";
-        }
+    # Soft-fail if command execution fails on sle 15
+    if (is_sle && sle_version_at_least '15') {
+        record_soft_failure 'https://fate.suse.com/323424';
+        select_console 'root-console';
+        script_run('SUSEConnect -s');
+        add_suseconnect_product("sle-module-desktop-applications");
+        zypper_call("ref");
```
Comment (okurz) [inline tests/console/command_not_found.pm]: to be consistent would you mind to use single ticks by default everywhere?

Comment (rwx788) [reply tests/console/command_not_found.pm]: Absolutely

**Comment by rwx788** [comment]
@Soulofdestiny, could you please review?

**Comment by okurz** [comment]
LGTM

I will leave the merging to @Soulofdestiny , don't know why you need his review but don't want to merge prematurely

**Comment by Soulofdestiny** [comment]
LGTM

**Pull request merged**